<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gestion_animation</class>
 <widget class="QDialog" name="Gestion_animation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>581</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestion programme d'animation</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>581</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ajouter EV</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>107</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;NOM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>177</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;DATE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>35</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NOMEV">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IDEV">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Valider">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Retour">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>390</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_DATREV">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;MAIL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_MAILEV">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;HEURE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_HEUREEV">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>233</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Consulter">
     <attribute name="title">
      <string>Consulter </string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>331</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="TRI">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TRI</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_RECH">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>300</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Supp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="SMS">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>SMS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EXCEL">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>EXCEL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Supprimer">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>350</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechEV">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher EV par DATE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_HEUREEV_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>243</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>45</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_DATREV_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background=#FFFFFF;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;HEURE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>187</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;DATE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NOMEV_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;MAIL&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Modifier">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>394</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>117</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;NOM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IDEV_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_MAILEV_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>310</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Mailling</string>
     </attribute>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>destinataire </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_pass">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>mail mot de passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sujet">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>561</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>sujet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseBtn">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Parcourir ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>431</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>pièce jointe ici </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>551</width>
        <height>271</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>tapez votre message ici ...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>561</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
